src/views/app/detail-panel/edit/edit-view.jsx

```diff
@@ -1053,6 +1082,7 @@ export default function EditView({ mailId, folderId, setHeader, toggleAppBoard }
 													editor?.text[1] ? `${editor.text[1]}${ev.target.value}` : ev.target.value
 												}`
 											];
+											saveToDraft();
 											updateSubjectField({ text: data });
 											onChange(data);
```

Comment:
saving on subject change seems not to be working as intended, moving a line down should solve the problem
```suggestion
											updateSubjectField({ text: data });
											onChange(data);
											saveToDraft();
```